subscription-fee/src/service.rs

```diff
@@ -1,15 +1,15 @@
 multiversx_sc::imports!();
 multiversx_sc::derive_imports!();
 
-use auto_farm::common::address_to_id_mapper::{AddressId, NULL_ID};
-
 use crate::common_storage;
 use crate::subtract_payments::Epoch;
 use crate::{fees, pair_actions};
 
 pub const MAX_USER_DEPOSITS: usize = 20;
 pub const MAX_SERVICES_LENGTH: usize = 20;
 
+pub const NULL_ID: AddressId = 0;
+
 #[derive(TypeAbi, TopEncode, TopDecode, NestedEncode, NestedDecode, ManagedVecItem)]
 pub struct ServiceInfo<M: ManagedTypeApi> {
     pub opt_payment_token: Option<TokenIdentifier<M>>,
```

Comment:
No, it can be empty. We check if it is accepted in case we have Some(value). Otherwise we save the none value, as this means we can accept whatever token, and then the logic applies on a pegged usd value (in subtractPayment, we have `subtract_specific_token` if Some, or `subtract_any_token` if None).